### PR TITLE
Add a prompt message when using self roles

### PR DESCRIPTION
**Pull request by NotReallyEight**
**Changes this PR makes:**
- Add a prompt message when using self roles
**Images**
image
image
image

@DTrombett Hi :heart:

### CODE REVIEW

**Comment by DTrombett** [comment]
PR farming alert

**Comment by NotReallyEight** [comment]
NO.

**Comment by NotReallyEight** [comment]
I'm just working. :sparkles:

File: src/events/componentEvents/notification-roles-select.ts [hunk 1]
```diff
@@ -8,16 +8,25 @@ export const event = new ComponentEvent(
...
 				.map((r) => r.id)
 				.filter((r) => !roleIds.includes(r));
+			const addedRoles: string[] = [];
 
 			if (interaction.values.length)
-				interaction.values.forEach((v) => newRoles.push(v.split("-")[2]));
+				interaction.values.forEach((v) => {
+					newRoles.push(v.split("-")[2]);
+					addedRoles.push(`<@&${v.split("-")[2]}>`);
+				});
```
Comment (DTrombett) [inline src/events/componentEvents/notification-roles-select.ts:22]: No, please!
```suggestion
				for (v of interaction.values) {
					newRoles.push(v.split("-")[2]);
					addedRoles.push(`<@&${v.split("-")[2]}>`);
				}
```

File: src/events/componentEvents/pronoun-roles-select.ts [hunk 2]
```diff
@@ -8,16 +8,25 @@ export const event = new ComponentEvent(
...
 				.map((r) => r.id)
 				.filter((r) => !roleIds.includes(r));
+			const addedRoles: string[] = [];
 
 			if (interaction.values.length)
-				interaction.values.forEach((v) => newRoles.push(v.split("-")[2]));
+				interaction.values.forEach((v) => {
+					newRoles.push(v.split("-")[2]);
+					addedRoles.push(`<@&${v.split("-")[2]}>`);
+				});
```
Comment (DTrombett) [inline src/events/componentEvents/pronoun-roles-select.ts:22]: No, please! (1)
```suggestion
				for (v of interaction.values) {
					newRoles.push(v.split("-")[2]);
					addedRoles.push(`<@&${v.split("-")[2]}>`);
				}
```

**Review by DTrombett — CHANGES_REQUESTED** [review]
Oof

File: src/events/componentEvents/notification-roles-select.ts (hunk 1 above)
Comment (NotReallyEight) [reply src/events/componentEvents/notification-roles-select.ts:22]: ...

**Comment by NotReallyEight** [comment]
Should be good now...

**Review by DTrombett — APPROVED** [review]
🚀

**Pull request merged**